data-prepper-api/src/main/java/org/opensearch/dataprepper/model/configuration/SinkModel.java

```diff
@@ -46,6 +48,15 @@ public Collection<String> getRoutes() {
         return this.<SinkInternalJsonModel>getInternalJsonModel().routes;
     }
 
+    public List<String> getIncludeKeys() {
```

Comment:
This can be a `Collection<String>` since the order doesn't matter to users. It should help add clarity when authors create new sinks.


```
public Collection<String> getIncludeKeys()
```